test/server/_fakes/data/services/singleApplication.js

```diff
@@ -0,0 +1,21 @@
+const serviceData = {
+  singleApp: {
+    getApplication: {
+      kind: 'application#application',
+      links: {
+        self: {
+          href: 'http://chs-dev:5004/applications/5f367ae40306d24a83d27985'
+        }
+      }
+    },
+    etag: 'Etag Not Implemented',
+    id: '5f4533b61d33fb291642665c',
+    name: 'app',
+    description: 'This application is used for blah',
+    privacy_policy_url: 'https://app.domain/privacy-policy',
+    terms_and_conditions_url: 'https://app.domain/terms-and-conditions',
+    creator_id: 'Users are not implemented',
+    owner_id: 'Users are not implemented'
+  }
+};
+module.exports = serviceData;
```

Comment:
It doesn't need to be changed here we will do that in the unit test branch, but we found out on Friday that this format isn't correct

Reply:
Happy to leave it to be fixed on the test branch then.